packages/cli/src/eventbus/middleware/logStreamingEnabled.middleware.ee.ts

```diff
@@ -1,5 +1,5 @@
 import type { RequestHandler } from 'express';
-import Container from 'typedi';
+import { Container } from 'typedi';
```

Comment:
They both resolve to the same class, why is this import style preferable? And if this is better, let's do it everywhere else as well? I find many with `import Container`.

Reply:
there is no difference. Just trying to make these imports look consistent.
If you prefer `import Container` over `import { Container }`, then let's change all the imports to use the first format.

Reply:
No strong preference, I was wondering if I was missing a difference. This PR is big enough, let's keep it as is for now.